packages/react-form-state/src/FormState.ts

```diff
@@ -70,16 +72,31 @@ export default class FormState<
 
   static getDerivedStateFromProps<T>(newProps: Props<T>, oldState?: State<T>) {
     const newInitialValues = newProps.initialValues;
+    const newExternalErrors = newProps.externalErrors || [];
 
     if (oldState == null) {
```

Comment:
Does oldState ever equal to undefined or null?
https://github.com/Shopify/quilt/blob/db4eed7982cda2581ea71251f1e49afc2b8b4463/packages/react-form-state/src/FormState.ts#L105

Reply:
it doesn't, if this gets rebased this line is gone in latest master.